setup.cfg

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python                                                           
+# -*- coding: utf-8 -*-                                                         
+#                                                                               
+# Copyright (c) 2019 VMware, Inc. All Rights Reserved.                          
+# SPDX-License-Identifier: BSD-2-Clause
+
+[metadata]
+name = tern
+author = VMWare Inc
+author-email = nishak@vmware.com
+summary = A compliance tool for containers
```

Comment:
"A container inspection tool for OSS license compliance"
Is there a character limit for this?

Reply:
It was originally:
```
description=("An inspection tool to find the OSS compliance metadata of"
                      " the packages installed in a container image."),
```

Reply:
I couldn't figure out how to wrap it but also didn't give it much time. Let me look more in to this.

Reply:
Since there is no linting on this file, we can put the entire description on one line.